plugins/uefi-dbx/fu-uefi-dbx-plugin.c

```diff
@@ -108,6 +111,14 @@ fu_uefi_dbx_plugin_constructed(GObject *obj)
 		/* TODO figure out non-snap scenarios */
 		g_info("snapd integration outside of snap is not supported");
 	}
+
+	/* ensure that an ESP was found */
+	esp = fu_context_get_default_esp(fu_plugin_get_context(plugin), &error_udisks2);
```

Comment:
I feel like we should either be passing NULL for the GError, or printing it as a g_info or something.

Reply:
done